tests_e2e/test_e2e.py

```diff
@@ -0,0 +1,102 @@
+import os
+from dotenv import load_dotenv, find_dotenv
+from threading import Thread
+from selenium import webdriver 
+import pytest
+import requests
+from todo_app import app
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.remote.webelement import WebElement
+import time
+import todo_app.trello_client as trello_client
+
+@pytest.fixture(scope="module") 
+def driver():
+    with webdriver.Firefox(executable_path='/Users/bacis/Documents/Corndel/Module-3/DevOps-Course-Starter/geckodriver') as driver:
```

Comment:
This is obviously specific to your machine. Best would be to change this to just 
```suggestion
    with webdriver.Firefox() as driver:
```

And add your geckodriver to your PATH

Reply:
done